### ISSUE TITLE
Third column missing

### ISSUE THREAD

**Issue by B4rc1** [sudormrfbin/cheatsheet.nvim#6]
I just wanted to give this plugin a quick try, but on my system, the third column is missing:

![grafik](https://user-images.githubusercontent.com/10383737/123521401-a96fcb00-d6b6-11eb-8bd4-68367fdde911.png)
---
relevant part of my config:
```lua
use {
  'sudormrfbin/cheatsheet.nvim',
  requires = {
    {'nvim-telescope/telescope.nvim'},
    {'nvim-lua/popup.nvim'},
    {'nvim-lua/plenary.nvim'},
  },
  config = function ()
    mapleader("hh", ":Cheatsheet<CR>")
  end
}
```
---
my telescope config (in case thats relevant):
```lua
use {
    'nvim-telescope/telescope.nvim',
    requires = {{'nvim-lua/popup.nvim'}, {'nvim-lua/plenary.nvim'}},
    config = function()

    local actions = require('telescope.actions')

    require('telescope').setup{
      defaults = {
        vimgrep_arguments = {
          'rg',
          '--color=never',
          '--no-heading',
          '--with-filename',
          '--line-number',
          '--column',
          '--smart-case'
        },
        prompt_position = "bottom",
        prompt_prefix = "> ",
        selection_caret = " ",
        entry_prefix = "  ",
        initial_mode = "insert",
        selection_strategy = "reset",
        sorting_strategy = "descending",
        layout_strategy = "center",
        layout_defaults = {
          horizontal = {
            mirror = false,
          },
          vertical = {
            mirror = false,
          },
        },
        file_sorter =  require'telescope.sorters'.get_fuzzy_file,
        file_ignore_patterns = {},
        generic_sorter =  require'telescope.sorters'.get_generic_fuzzy_sorter,
        shorten_path = true,
        winblend = 0,
        width = 0.55,
        preview_cutoff = 120,
        results_height = 15,
        results_width = 0.8,
    		results_title = false,
        border = {},
    		borderchars = {
          { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
          prompt = {"─", "│", " ", "│", "╭", "╮", "│", "│"},
          results = {"─", "│", "─", "│", "├", "┤", "╯", "╰"},
          preview = { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
        },
        color_devicons = true,
        use_less = true,
        set_env = { ['COLORTERM'] = 'truecolor' }, -- default = nil,
        file_previewer = require'telescope.previewers'.vim_buffer_cat.new,
        grep_previewer = require'telescope.previewers'.vim_buffer_vimgrep.new,
        qflist_previewer = require'telescope.previewers'.vim_buffer_qflist.new,
    		mappings = {
    			i = {
    				["C-j"] = actions.move_selection_next,
    				["C-k"] = actions.move_selection_previous,
    			},
    			n = {
    				["C-c"] = actions.close,
    			}
    		},

        -- Developer configurations: Not meant for general override
        buffer_previewer_maker = require'telescope.previewers'.buffer_previewer_maker
      }
    }

    -- Telescope
    function search_nvim_config()
        require("telescope.builtin").find_files(
            {
                prompt_title = "Neovim Config",
                cwd = "$HOME/.config/nvim/"
            }
        )
    end


    map("n", "z=", "<Cmd>:Telescope spell_suggest<CR>", { noremap = true, silent = true })
    map("n", "gr", "<Cmd>:Telescope lsp_references<CR>", { noremap = true, silent = true })

    mapleader("<space>", "<Cmd>:Telescope find_files<CR>")
    mapleader("?", "<Cmd>:Telescope keymaps<CR>")
    mapleader("h:", "<Cmd>:Telescope help_tags<CR>")
    mapleader("bb", "<Cmd>:Telescope buffers<CR>")
    mapleader("ff", "<Cmd>:Telescope file_browser<CR>")
    mapleader("fg", "<Cmd>:Telescope live_grep<CR>")
    mapleader("fc", "<Cmd>:lua search_nvim_config()<CR>")

    end
}
```
----

link to my entire config, in case it's something else: https://codeberg.org/B4rc1/nvim-config (does not have the cheatsheet snippet i posted here, because I don't (yet ;)) want it on my other machines)

----
`nvim --version`
```
NVIM v0.5.0-dev+1447-g5c0b01fd8
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions     -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security     -fstack-clash-protection -fcf-protection -Wp,-U_FORTIFY_SOURCE -Wp,-D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/jonas/.cache/paru/clone/neovim-git/src/build/config -I/home/jonas/.cache/paru/clone/neovim-git/src/neovim-git/src -I/usr/include -I/home/jonas/.cache/paru/clone/neovim-git/src/build/src/nvim/auto -I/home/jonas/.cache/paru/clone/neovim-git/src/build/include
```

**Comment by sudormrfbin** [comment]
Hi !

Does the command line get filled with `:%d` when you search for `Cut all` and hit enter ?

**Comment by B4rc1** [comment]
yes, it does

**Comment by sudormrfbin** [comment]
I was using the deprecated `results_width` to calculate width of each column instead of `width`. Should be fixed now.

**Comment by B4rc1** [comment]
Well now I get an error when trying to open the cheatsheet:
```
Error detected while processing function <SNR>41_Cheatsheet:
line    5:
E5108: Error executing lua ...acker/start/cheatsheet.nvim/lua/cheatsheet/telescope.lua:42: attempt to perform arithmetic on local 'width' (a nil value)
```

**Comment by B4rc1** [comment]
ah, I think I know why. width has been moved from `opts.width` to `layout_config.width` in a recent telescope [commit](https://github.com/nvim-telescope/telescope.nvim/commit/5a53ec5c2fdab10ca8775d3979b1a85e63d57953)

**Comment by sudormrfbin** [comment]
> ah, I think I know why. width has been moved from `opts.width` to `layout_config.width` in a recent telescope [commit](https://github.com/nvim-telescope/telescope.nvim/commit/5a53ec5c2fdab10ca8775d3979b1a85e63d57953)

Thanks for the pointer :D. I pushed a fix to 803d62d0de9ce2c84603f22f8b48996691bda025, can you check if it's working now ?

**Comment by B4rc1** [comment]
Works like a charm now, thanks!

![grafik](https://user-images.githubusercontent.com/10383737/124349997-43bd9a80-dbf2-11eb-83f3-bc9af415a250.png)
